### DISCUSSION TITLE
Compiled scripts?

### DISCUSSION THREAD

**Discussion by SpecialCharacter** [espanso/espanso#1221] [Feature requests and Ideas]
I am thinking of users who have no admin right to install expanso. Is it possible to generate compiled individual scripts? I.e., .exe and whatever the equivalent is on Mac and Linux?

**Comment by federico-terzi** [comment]
Hey @SpecialCharacter,

Thank you for the feedback! I think this could be possible in theory on Linux and Windows, but not on macOS, as there you need admin rights to enable the Accessibility API.
In the meanwhile, you might also get away with portable mode (once it gets implemented) #623

Cheers :)
